.github/workflows/pages.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy docs to Pages
+
+on:
+  push:
+    branches: ['master']
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
```suggestion
        uses: actions/checkout@v4
```

Reply:
This and the ruby version are from the parent PR - I don't see whyy not, so I update it.